scripts/variantstore/wdl/GvsQuickstartHailIntegration.wdl

```diff
@@ -7,11 +7,13 @@ import "GvsQuickstartVcfIntegration.wdl" as QuickstartVcfIntegration
 workflow GvsQuickstartHailIntegration {
     input {
         String branch_name
+        Boolean is_wgs = true
         Boolean use_VQSR_lite = true
         File interval_list
         Boolean use_classic_VQSR = true
         Boolean extract_do_not_filter_override
         String dataset_suffix = "hail"
+        Boolean use_default_dockers = false
```

Comment:
Is this `use_default_dockers` logic duplicated in the Hail and VCF integration WDLs so we can run them separate of the top-level WDL?

Reply:
um. I think.